src/components/PollData.js

```diff
@@ -30,27 +42,56 @@ const PollData = (props) => {
             { cancelable: false }
         );
 
+    const updatePollTitle = (title, id) => {
+        showModal(true)
+        const arr = props.pollData.filter((e) => e._id == id)
```

Comment:
use find instead of filter.
then no need access arr[0]
it will return the object